### PR TITLE
bugfix/PLAYER-5242 Player controls are not displayed on fullscreen

### PR DESCRIPTION
**Pull request by KseniiaSokolovaEpam**
There was an checking if(this.isMounted ()) {...}, but function "isMounted" never existed, so when the window was resized (switching to fullscreen mode), the function was called and there was an error.

Unit tests passed.

### CODE REVIEW

File: js/views/adScreen.jsx [hunk 1]
```diff
@@ -15,6 +15,8 @@ import Utils from '../components/utils';
  * The screen to be displayed on advertising
  */
 class AdScreen extends React.Component {
+  isMounted = false;
```
Comment (ooyala-amohnatkin) [inline js/views/adScreen.jsx]: This is obsolete. Component will not receive new prop if it's unmounted

Comment (ooyala-amohnatkin) [reply js/views/adScreen.jsx]: also replace `componentWillReceiveProps` with `componentDidUpdate`

Comment (KseniiaSokolovaEpam) [reply js/views/adScreen.jsx]: Thank you Alexey! @ooyala-amohnatkin

File: tests/views/adScreen-test.js [hunk 2]
```diff
@@ -270,13 +271,44 @@ describe('AdScreen', function() {
...
+    expect(controlBarVisible).toBe(true);
+
+    const node = document.createElement('div');
+    Enzyme.mount(
+      <AdScreen
+        controller={mockController}
+        skinConfig={mockSkinConfig}
+        componentWidth={400}
+        fullscreen={false}
+      />, node
```
Comment (ooyala-amohnatkin) [inline tests/views/adScreen-test.js]: `mount` doesn't take second argument

File: tests/views/adScreen-test.js [hunk 3]
```diff
@@ -270,13 +271,44 @@ describe('AdScreen', function() {
...
+    );
+
+    Enzyme.mount(
```
Comment (ooyala-amohnatkin) [inline tests/views/adScreen-test.js]: As per documentation, it unmounts component and mounts it again, so you don't simulate resize here.
Please use `setProps` to update width

File: tests/views/adScreen-test.js [hunk 4]
```diff
@@ -270,13 +271,44 @@ describe('AdScreen', function() {
...
       />);
-    var unmuteIcons = wrapper.find(UnmuteIcon);
+    const unmuteIcons = wrapper.find(UnmuteIcon);
     expect(unmuteIcons.length).toBe(0);
   });
 
+  it('should hide controlBar after resize', () => {
+    let controlBarVisible = true;
+
+    mockController.startHideControlBarTimer = () => {
```
Comment (ooyala-amohnatkin) [inline tests/views/adScreen-test.js]: would be great to use `jest.fn()` instead and change the expectation to `expect(mockController.startHideControlBarTimer).toBeCalled()`;

File: tests/views/adScreen-test.js [hunk 5]
```diff
@@ -270,13 +271,44 @@ describe('AdScreen', function() {
...
+        fullscreen={false}
+      />, node
+    );
+
+    Enzyme.mount(
+      <AdScreen
+        controller={mockController}
+        skinConfig={mockSkinConfig}
+        componentWidth={800}
+        fullscreen={true}
```
Comment (ooyala-amohnatkin) [inline tests/views/adScreen-test.js]: `startHideControlBarTimer` is called here just because `fullscreen` is set to `true`

Comment (KseniiaSokolovaEpam) [reply tests/views/adScreen-test.js]: @ooyala-amohnatkin :-) yes. you are right

File: tests/views/adScreen-test.js (hunk 4 above)
Comment (KseniiaSokolovaEpam) [reply tests/views/adScreen-test.js]: \+

File: tests/views/adScreen-test.js (hunk 3 above)
Comment (KseniiaSokolovaEpam) [reply tests/views/adScreen-test.js]: \+

File: tests/views/adScreen-test.js (hunk 2 above)
Comment (KseniiaSokolovaEpam) [reply tests/views/adScreen-test.js]: \+

**Pull request merged**